app/components/doi-citation-body.js

```diff
@@ -0,0 +1,18 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+  tagName: 'div',
+
+  didReceiveAttrs() {
+    this._super(...arguments);
+
+  },
+
+  didRender() {
+    this._super(...arguments);
+
+    window.MathJax.typesetClear([this.get('element')]);
```

Comment:
Because the MathJax script is loaded async, is there a chance that this (and other window.MathJax) lines could get called before it's loaded? Would that cause issues?

Reply:
Great point.  I am checking to see what ember offers for this.  The library is on the large side, so I am guessing that loading synchronously would not be the answer of choice.